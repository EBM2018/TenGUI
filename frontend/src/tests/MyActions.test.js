import MyActions from '../react/Dashboard/MyActions/MyActions';
import React from 'react';
import renderer from 'react-test-renderer';


test ('derp',()=>{
    expect('True').toBeTruthy;
});
test('does this render', ()=> {
    const component=renderer
        .create(<MyActions/>);
    let tree=component.toJSON();
    expect(tree).toMatchSnapshot();
});