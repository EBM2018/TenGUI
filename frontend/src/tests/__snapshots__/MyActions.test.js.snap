// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`does this render 1`] = `
Array [
  <h2
    className="is-h2"
  >
    Mes Actions
  </h2>,
  <div
    className="column is-5 has-text-left"
  >
    <p
      className="control has-icons-left has-icons-right"
    >
      <button
        className="button is-info is-inverted is-fullwidth"
        name="button"
        type="button"
      >
        Vous avez compris ?
      </button>
      <span
        className="icon is-small is-left"
      >
        <i
          className="fas fa-brain"
        />
      </span>
    </p>
  </div>,
  <div
    className="column is-5 has-text-left"
  >
    <p
      className="control has-icons-left has-icons-right"
    >
      <button
        className="button is-info is-inverted is-fullwidth"
        name="button"
        type="button"
      >
        Faites un résumé ?
      </button>
      <span
        className="icon is-small is-left"
      >
        <i
          className="fas fa-file"
        />
      </span>
    </p>
  </div>,
  <div
    className="column is-5 has-text-left"
  >
    <p
      className="control has-icons-left has-icons-right"
    >
      <button
        className="button is-info is-inverted is-fullwidth"
        name="button"
        type="button"
      >
        Vous en êtes où ?
      </button>
      <span
        className="icon is-small is-left"
      >
        <i
          className="fas fa-clipboard-list"
        />
      </span>
    </p>
  </div>,
  <div
    className="column is-5 has-text-left"
  >
    <p
      className="control has-icons-left has-icons-right"
    >
      <button
        className="button is-info is-inverted is-fullwidth"
        name="button"
        type="button"
      >
        Reboost
      </button>
      <span
        className="icon is-small is-left"
      >
        <i
          className="fas fa-flushed"
        />
      </span>
    </p>
  </div>,
]
`;
